quickwit/quickwit-metastore/migrations/postgresql/8_add-split-indexing-end-timestamp-field.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE splits
+    ADD COLUMN indexing_end_timestamp TIMESTAMP NOT NULL DEFAULT (CURRENT_TIMESTAMP AT TIME ZONE 'UTC');
```

Comment:
We could avoid all those Postgres conversions `INTERVAL '{} seconds'` or `to_timestamp(time_range_end)` by using `BIGINT` type. We then use  `extract(epoch from now())` to extract the current timestamp for comparison. Should we do that?

Reply:
I would not make this change now and remain consistent with other timestamps.

Reply:
Ok, I was just talking in terms of simplifying.